website/docs/r/catalog_media.html.markdown

```diff
@@ -42,7 +47,8 @@ resource "vcd_catalog_media" "myNewMedia" {
 The following arguments are supported:
 
 * `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
-* `catalog` - (Required) The name of the catalog where to upload media file
+* `catalog` - (Optional; Deprecated) The name of the catalog where to upload media file
+* `catalog_id` - (Optional; *v3.8.2+*) The ID of the catalog where to upload media file
```

Comment:
Should we mention that despite being optional, either `catalog` or `catalog_id` is required?

Reply:
done